v2/preprocess_data.py

```diff
@@ -0,0 +1,258 @@
+# PURPOSE: This program preprocesses existing data for machine learning training.
```

Comment:
lets rename it to `user_reports_preprocessor.py`? We might have other jobs in future so was thinking this name might be very generic?

Reply:
Sure.